crates/router/src/connector/braintree/braintree_graphql_transformers.rs

```diff
@@ -47,6 +52,12 @@ pub struct TransactionBody {
     merchant_account_id: Secret<String>,
 }
 
+impl ConnectorCurrencyCommon for PaymentsAuthorizeData {
```

Comment:
Please add this in script also, Let's implement ConnectorCurrencyCommon without get_currency_unit impl in the script